mscore/musescore.cpp

```diff
@@ -441,7 +441,7 @@ void updateExternalValuesFromPreferences() {
 //   preferencesChanged
 //---------------------------------------------------------
 
-void MuseScore::preferencesChanged(bool fromWorkspace)
+void MuseScore::preferencesChanged(bool fromWorkspace, bool changeUI)
```

Comment:
I think the name should be `changedUI` or `changeUIPref`

Reply:
No. This is the flag that tells whether the method need to actually change UI or not.
Common policy for boolean flags in the form of verb.

Reply:
Strictly speaking, not in this case :)
Method name `preferencesChanged` - preferences have been changed.
What kind? 
ui preferences have been changed - `uiPrefChanged`

It doesn’t matter

By the way, the method would have to be called `onPreferencesChanged`